### PR TITLE
bcftools 1.5

### PR DESCRIPTION
**Pull request by tseemann**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [ ] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### Updates to existing formula: have you

- [x] Removed the `revision` line, if any, when bumping the version number?
- [ ] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [x] Not altered the `bottle` section?
- [x] Checked that the tests still pass?

### CODE REVIEW

**Comment by tseemann** [comment]
Ok, i figured out they have moved to `automake`. I am working on it.

**Comment by tseemann** [comment]
```
The command "sudo apt-get install -y libxml-parser-perl libxml-sax-perl" failed and exited with 100 during .
```

File: bcftools.rb [hunk 1]
```diff
@@ -17,19 +18,28 @@ class Bcftools < Formula
 
   deprecated_option "with-polysomy" => "with-gsl"
 
+  fails_with :clang
```
Comment (jonchang) [inline bcftools.rb]: Please add a reason why this formula fails with `clang` e.g.,

```ruby
fails_with :clang do
  cause "clang doesn't foo the bar"
end
```

File: bcftools.rb [hunk 2]
```diff
@@ -17,19 +18,28 @@ class Bcftools < Formula
...
+  fails_with :llvm
+
```
Comment (jonchang) [inline bcftools.rb]: homebrew/science/bcftools:
  * C: 23: col 1: Trailing whitespace detected.
  * 'fails_with :llvm' is now a no-op so should be removed

File: bcftools.rb [hunk 3]
```diff
@@ -17,19 +18,28 @@ class Bcftools < Formula
...
   depends_on "bzip2" unless OS.mac?
 
   def install
-    args = build.with?("gsl") ? "USE_GPL=1" : []
-    system "make", "all", "install", "prefix=#{prefix}", *args
+    args = build.with?("gsl") ? "--enable-libgsl" : []
+    system "./configure",
+           "--prefix=#{prefix}",
+           "--with-htslib=#{Formula["htslib"].opt_prefix}",
+           *args
```
Comment (jonchang) [inline bcftools.rb]: Restructure this to something like

```ruby
args = %W[ ... ]
args << "--enable-libgsl" if build.with? "gsl"
system "./configure", *args
```

**Comment by tseemann** [comment]
Thanks for the excellent feedback @jonchang - correcting now.

**Comment by tseemann** [comment]
I am confused
```
Error: No available formula with the name "Calling fails_with :llvm is deprecated!" 
```

I remove the `fails_with :llvm` (and clang) I thought?
Have I screwed up my rebase?

**Comment by jonchang** [comment]
Yeah you have to rebase and force push I think

**Comment by iMichka** [comment]
Yes rebasing on master will fix that.

**Comment by tseemann** [comment]
I messed this up, my git fu is not good enough. I will re-submit a new PR.

**Pull request closed without merging**